docs/openapi/components/schemas/common/PestSample.yml

```diff
@@ -0,0 +1,240 @@
+$linkedData:
+  term: PestSample
+  '@id': https://w3id.org/traceability#PestSample
+title: Pest Sample
+description: Information regarding a sample of some indeterminate pest.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - PestSample
+    default:
+      - PestSample
+    items:
+      type: string
+      enum:
+        - PestSample
+  hostName:
+    title: Host Name
+    description: The scientific name for the plant or plant product. Should contain at minimum genus and species.
+    $ref: ./Taxonomy.yml
+    $linkedData:
+      term: hostName
+      '@id': https://w3id.org/traceability#Taxonomy
+  hostQuantity:
+    title: Host Quantity
+    description: The quantity and units of host plants. Shipment size (in e.g. "Cartons" or "Cubic Meters" for cargo interceptions), acres or individual plants for growing site interceptions).
+    $ref: ./QuantitativeValue.yml
+    $linkedData:
+      term: hostQuantity
+      '@id': http://rs.tdwg.org/dwc/terms/organismQuantity
+  affected:
+    title: Affected
+    description: The percentage of hosts affected, if available.
+    type: number
+    $linkedData:
+      term: affected
+      '@id': https://dwc.tdwg.org/list/#dwc_measurementValue
+  plantDistribution:
+    title: Plant Distribution
+    description: >-
+      The distribution of host plant (for cargo interceptions leave blank). Must be one of:
+      Limited, Scattered, Widespread.
+    type: string
+    $linkedData:
+      term: plantDistribution
+      '@id': http://rs.tdwg.org/dwc/terms/degreeOfEstablishment
+  plantPartsAffected:
+    title: Plant Parts Affected
+    description: >-
+      A list of plant parts affected (for cargo interceptions this will be limited to harvested
+      parts). Must be one of: Leaves, Upper Surface; Leaves, Lower Surface; Petiole; Stem;
+      Trunk/Bark; Branches; Growing Tiips; Roots; Bulbs, Tubers, Corms; Buds; Flowers; Fruits or
```

Comment:
```suggestion
      A list of plant parts affected (for cargo interceptions, this will be limited to harvested
      parts). Must be one of: Leaves, Upper Surface; Leaves, Lower Surface; Petiole; Stem;
      Trunk/Bark; Branches; Growing Tips; Roots; Bulbs, Tubers, Corms; Buds; Flowers; Fruits or
```